packages/terra-base/src/Base.scss

```diff
@@ -11,6 +11,7 @@ html {
 
 body {
   background-color: var(--terra-base-background-color, #fff);
+  background-image: var(--terra-base-background-image, none);
```

Comment:
[Background image needs to go on the html element](https://github.com/cerner/terra-core/issues/1200), I've seen issues using CSS radial gradients as background-image property on body element.

Let's move the background color to the html element as well.

```css
html {
  background-color: var(--terra-base-background-color, #fff);
  background-image: var(--terra-base-background-image, none);
}
```